pkl/evaluator.go

```diff
@@ -81,7 +81,10 @@ func (e *evaluator) EvaluateModule(ctx context.Context, source *ModuleSource, ou
 func (e *evaluator) EvaluateOutputText(ctx context.Context, source *ModuleSource) (string, error) {
 	var out string
 	err := e.EvaluateExpression(ctx, source, "output.text", &out)
-	return out, err
+	if err != nil {
+		return "", err
+	}
+	return out, nil
```

Comment:
This is effectively the same business logic (in `EvaluateExpression, if `err` exists, `out` is an empty string)